index.html

```diff
@@ -303,6 +334,27 @@ <h2>Other links</h2>
         </li>
       </ul>
     </section>
+    <section>
+      <h2>RFCs and other relevant documents</h2>
```

Comment:
Maybe make this “Related standards and document formats” and also include the OGC link? (Or call out specific OGC standards even)  And probably move it up before the general “Other Links” section?

Reply:
I put the OGC standards in the renamed section as well.